src/components/pages/Navbar/NavbarFeatures/SignupButton.js

```diff
@@ -3,19 +3,21 @@ import { useAuth0 } from '@auth0/auth0-react';
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function SignupButton() {
-  const { loginWithRedirect } = useAuth0();
+  const { loginWithRedirect, isAuthenticated } = useAuth0();
 
   return (
-    <NavBtnLink
-      className="btn btn-primary btn-block"
-      onClick={() =>
-        loginWithRedirect({
-          screen_hint: 'signup',
-        })
-      }
-    >
-      Signup
-    </NavBtnLink>
+    !isAuthenticated && (
+      <NavBtnLink
+        className="btn btn-primary btn-block"
+        onClick={() =>
+          loginWithRedirect({
+            screen_hint: 'signup',
+          })
+        }
+      >
+        Signup
+      </NavBtnLink>
+    )
   );
 }
 
```

Comment:
It's looking like the updated 'isAuthenticated' work with the AuthO updates Elijah, Farhaan, and Mike have been working on here. Unless I'm reading it wrong and need more coffee... But if that's the case I'd suggest an extra story point or two be added just for the forward thinking way you set this up.